### PR TITLE
added ability to pass in logger

### PR DESCRIPTION
**Pull request by onetwopunch**
This should fix issue #5 

Now you can pass in any arbitrary object that responds to debug info warn error and fatal as the logger. As an example, you can now call:

    Logplexer.error( "Will log to Rails logger", logger: Rails.logger )

### CODE REVIEW

**Comment by onetwopunch** [comment]
@8bitDesigner Could you take a look at this and make sure it fulfills your requirements?

File: lib/logplexer.rb [hunk 1]
```diff
@@ -56,6 +64,11 @@ def log( exception, log_level, opts = {})
     end
   end
 
+  def is_logger?(logger)
+    eval( %w(debug info warn error fatal).map do |meth|
+      "logger.respond_to?(:#{meth})"
```
Comment (8bitDesigner) [inline lib/logplexer.rb]: 0.o

Comment (8bitDesigner) [inline lib/logplexer.rb]: OH! It's short for METHOD!

Comment (onetwopunch) [inline lib/logplexer.rb]: Lol yeah about that :syringe: :smiling_imp:

**Comment by 8bitDesigner** [comment]
This is close, but I'd love to configure this on a per env basis: Like:

(in `/config/environments/development.rb`
``` ruby
config.logplexer.logger = Rails.logger
```

**Comment by onetwopunch** [comment]
@8bitDesigner Good idea, I'll try to shim that into the Rails config.

**Comment by onetwopunch** [comment]
The ability to add global configuration has been added in [v1.0.0](https://github.com/Fullscreen/logplexer/tree/v1.0.0)

**Pull request merged**